src/test/kotlin/org/coepi/api/dao/ReportsDaoTest.kt

```diff
@@ -16,13 +22,49 @@ import java.util.*
  *
  * Until then, don't expect them to succeed, just use them as documentation
  */
+
 @Disabled
 class ReportsDaoTest {
 
     private val dao: ReportsDao = ReportsDao()
     val cenKeys = arrayOf("foo", "bar")
     val reportData = "foobar".toByteArray(Charset.defaultCharset())
 
+    companion object{
+
+        val LOCAL_URL= "http://localhost:9000";
+        val REGION = "us-west-2"
+
+        @BeforeAll
+        @JvmStatic
+        internal fun startDynamoDb(){
```

Comment:
Can we make this re-usable for other test classes can use it? One way is to create an abstract DaoTestBase class which provides a method for local table creation.

Please add the support for TCNReportsDao as well.

Reply:
Created a separate class to initialize dynamo db. Rather than an abstract class keeping it as a utility class.